### PR TITLE
[Docs] update Japanese translation of feature_unicode.md

### PR DESCRIPTION
**Pull request by umi-umi**
## Description

update Japanese translation of docs/feature_unicode.md.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [x] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by mtei** [comment]
@shelaf @s-show @hsgw @satt99 @takashiski @yoshihara よかったらレビューお願いします。

File: docs/ja/feature_unicode.md [hunk 1]
```diff
@@ -275,3 +277,22 @@ AutoHotkey のデフォルトのスクリプトで、カスタムホットキー
 
 システム上で米国インターナショナルレイアウトを有効にすると、文字にアクセントをつけるために区切り文字を使います。例えば、"\`a" は à になります。
 これを有効にする方法は[ここ](https://support.microsoft.com/en-us/help/17424/windows-change-keyboard-layout)で見つかります。
+
+## Linux でのソフトウェアキーボードレイアウト :id=custom-linux-layout
+
+この方法では、キーボード自体で Unicode をサポートする必要はありませんが、代わりに Xorg のカスタムキーボードレイアウトを使います。これは通常のキーボードで特殊文字が挿入される方法です。これは IBus を必要とせず、事実上すべてのソフトウェアで機能します。カスタムレイアウトの作成に関するヘルプは、[ここ](https://www.linux.com/news/creating-custom-keyboard-layouts-x11-using-xkb/)、[ここ](http://karols.github.io/blog/2013/11/18/creating-custom-keyboard-layouts-for-linux/)、[ここ](https://wiki.archlinux.org/index.php/X_keyboard_extension)にあります。`us` レイアウトを編集して、`RALT(KC_R)` で 🤣 を取得する方法の例:
```
Comment (s-show) [inline docs/ja/feature_unicode.md:283]: カスタムレイアウトの作成に関する3番目のヘルプですが、日本語訳がありましたので、訳注で案内するのはどうでしょうか。
```suggestion
この方法では、キーボード自体で Unicode をサポートする必要はありませんが、代わりに Xorg のカスタムキーボードレイアウトを使います。これは通常のキーボードで特殊文字が挿入される方法です。これは IBus を必要とせず、事実上すべてのソフトウェアで機能します。カスタムレイアウトの作成に関するヘルプは、[ここ](https://www.linux.com/news/creating-custom-keyboard-layouts-x11-using-xkb/)、[ここ](http://karols.github.io/blog/2013/11/18/creating-custom-keyboard-layouts-for-linux/)、[ここ](https://wiki.archlinux.org/index.php/X_keyboard_extension)にあります。（訳注：日本語の解説記事もあります -> [X KeyBoard extension - ArchWiki](https://wiki.archlinux.jp/index.php/X_KeyBoard_extension)）
`us` レイアウトを編集して、`RALT(KC_R)` で 🤣 を取得する方法の例:
```

File: docs/ja/feature_unicode.md [hunk 2]
```diff
@@ -131,6 +131,8 @@ QMK での Unicode の入力は、マクロのように、OS への一連の文
   デフォルトで有効になっていて、IBus が有効になったディストリビューションのほとんどどれでも動作します。IBus が無い場合、このモードは GTK アプリ下で動作しますが、他の場所ではほとんど動作しません。
   デフォルトでは、このモードは Unicode 入力を開始するために Ctrl+Shift+U (`LCTL(LSFT(KC_U))`) を使いますが、これは他のキーコードで [`UNICODE_KEY_LNX`](#input-key-configuration) を定義することで変更できます。これは、Ctrl+Shift+U の挙動が Ctrl+Shift+E に統合された IBus バージョン 1.5.15 以上を必要とするかもしれません。
 
+  IBus を使わない非 GTK アプリでのサポートを希望するユーザは、カスタムキーボードレイアウト([詳細はこちら](#custom-linux-layout))の作成など、より間接的な方法に頼る必要があるかもしれません。
```
Comment (shelaf) [inline docs/ja/feature_unicode.md:134]:
```suggestion
  IBus を使わずに非 GTK アプリでのサポートを希望するユーザは、カスタムキーボードレイアウト([詳細はこちら](#custom-linux-layout))の作成など、より間接的な方法に頼る必要があるかもしれません。
```

File: docs/ja/feature_unicode.md (hunk 1 above)
Comment (shelaf) [inline docs/ja/feature_unicode.md:283]:
```suggestion
この方法では、キーボード自体で Unicode をサポートする必要はありませんが、代わりに Xorg のカスタムキーボードレイアウトを使います。これは通常のキーボードで特殊文字が挿入される方法です。これは IBus を必要とせず、事実上すべてのソフトウェアで機能します。カスタムレイアウトの作成に関するヘルプは、[ここ](https://www.linux.com/news/creating-custom-keyboard-layouts-x11-using-xkb/)や[ここ](http://karols.github.io/blog/2013/11/18/creating-custom-keyboard-layouts-for-linux/)、[ここ](https://wiki.archlinux.org/index.php/X_keyboard_extension)にあります。`us` レイアウトを編集して、`RALT(KC_R)` で 🤣 を入力する方法の例:
```

File: docs/ja/feature_unicode.md [hunk 3]
```diff
@@ -275,3 +277,22 @@ AutoHotkey のデフォルトのスクリプトで、カスタムホットキー
...
+`xkb_symbols "basic" {` の中に、`include "level3(ralt_switch)"` を追加します。
+
+R キーを定義する行を見つけて、リストにエントリを追加し、以下のようにします:
+```
+key <AD04> {	[	  r,	R, U1F923		]	};
+```
+
+ファイルを保存し、コマンド `setxkbmap us` を実行して、レイアウトを再読み込みします。
+
+レイアウトで定義されたキーに1つのカスタム文字を定義し、4番目のレイヤーにデータを入力する場合は別のカスタム文字を定義できます。8番目までの追加レイヤーも可能です。
```
Comment (shelaf) [inline docs/ja/feature_unicode.md:296]:
```suggestion
レイアウトで定義されたキーに1つのカスタム文字を定義でき、4番目のレイヤーにデータを入力する場合は別のカスタム文字を定義できます。8番目までの追加レイヤーも可能です。
```

File: docs/ja/feature_unicode.md (hunk 1 above)
Comment (umi-umi) [reply docs/ja/feature_unicode.md]: 訳注、ありがとうございます。そのまま頂きました。

File: docs/ja/feature_unicode.md (hunk 2 above)
Comment (umi-umi) [reply docs/ja/feature_unicode.md]: ご指摘ありがとうございます。日本語は難しい。。

**Review by shelaf — APPROVED** [review]

**Review by peepeetee — APPROVED** [review]

**Comment by zvecr** [comment]
Closing as this would have gone stale multiple times by now.

**Pull request closed without merging**